docs/live_migration.md

```diff
@@ -127,3 +128,54 @@ src $ ch-remote --api-socket=/tmp/api send-migration unix:/tmp/sock
 
 When the above commands completed, the VM should be successfully
 migrated to the destination machine without interrupting the workload.
+
+### TCP Socket Migration
+
+If TCP socket is selected for migration, the cross-host migration
+performance will be further improved, but it is necessary to consider
```

Comment:
I'd drop the comment on performance differences, since we don't have much data to back it up. Also, the performance differences depend a lot on the end user environment. Let's leave it to users to evaluate and choose what's the best for their use cases.

Reply:
> I'd drop the comment on performance differences, since we don't have much data to back it up. Also, the performance differences depend a lot on the end user environment. Let's leave it to users to evaluate and choose what's the best for their use cases.

You are right, I will remove the description of performance here.